uk.co.compucorp.civicrm.hrleaveandabsences/views/shared/components/leave-calendar-legend.html

```diff
@@ -9,106 +9,22 @@ <h2 class="panel-title chr_leave-calendar__legend__title" role="button" ng-click
-              <span class="chr_leave-calendar__time-unit">10:00</span>
-            </div>
-            <span class="chr_leave-calendar__legend__label">Requested {{absenceType.title}} specific time</span>
-          </div>
+      <div class="chr_leave-calendar__legend__header">Types</div>
+      <div class="clearfix">
+        <div ng-repeat="absenceType in legend.absenceTypes"
+          class="chr_leave-calendar__legend__type-badge"
+          ng-style="legend.getAbsenceTypeStyle(absenceType)">
+          {{absenceType.title}}
         </div>
       </div>
-      <div class="row" ng-repeat-end ng-if="absenceType.allow_accruals_request == 1">
-        <div class="col-xs-6 col-md-3">
-          <div class="chr_leave-calendar__legend__entry">
-            <div class="chr_leave-calendar__item chr_leave-calendar__item--alt"
-               ng-style="{borderColor: absenceType.color}">
-              <span>AT</span>
-            </div>
-            <span class="chr_leave-calendar__legend__label">Accrued TOIL</span>
-          </div>
-        </div>
-        <div class="col-xs-6 col-md-3">
-          <div class="chr_leave-calendar__legend__entry">
-            <div class="chr_leave-calendar__item chr_leave-calendar__item--alt chr_leave-calendar__item--toil-requested"
-               ng-style="{borderColor: absenceType.color}">
-              <span>AT</span>
-            </div>
-            <span class="chr_leave-calendar__legend__label">Requested Accrued TOIL</span>
-          </div>
-        </div>
-      </div>
-      <div class="row">
-        <div class="col-xs-6 col-md-3">
-          <div class="chr_leave-calendar__legend__entry">
-            <div class="chr_leave-calendar__item chr_leave-calendar__day-container--type--weekend"></div>
-            <span class="chr_leave-calendar__legend__label">Weekend</span>
-          </div>
-        </div>
-        <div class="col-xs-6 col-md-3">
-          <div class="chr_leave-calendar__legend__entry">
-            <div class="chr_leave-calendar__item chr_leave-calendar__day-container--type--public-holiday"></div>
-            <span class="chr_leave-calendar__legend__label">Public Holiday</span>
-          </div>
-        </div>
-        <div class="col-xs-6 col-md-3">
-          <div class="chr_leave-calendar__legend__entry">
-            <div class="chr_leave-calendar__item chr_leave-calendar__day-container--type--non-working"></div>
-            <span class="chr_leave-calendar__legend__label">Non Working Day</span>
-          </div>
+      <div class="chr_leave-calendar__legend__header">Other</div>
+      <div class="clearfix">
+        <div ng-repeat="otherBadge in legend.other_badges"
+          class="chr_leave-calendar__legend__other-badge-wrapper">
+          <div class="chr_leave-calendar__legend__other-badge
+            {{otherBadge.type ? 'chr_leave-calendar__legend__other-badge-' + otherBadge.type : ''}}">
+            {{otherBadge.label}}</div>
+          {{otherBadge.hint}}
```

Comment:
The indentation here is not correct, it should move right.

Reply:
I don't think so. `{{otherBadge.hint}}` is a part of the parent elem.